ted/src/main/java/frc/robot/common/DriveTrainConfig.java

```diff
@@ -0,0 +1,109 @@
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// FRC - Crescendo 2024
+// File: DrivetrainConfig.java
```

Comment:
So this class essentially already exists.  See: frc.robot.common.DrivetrainConfig

Reply:
so seems like its the casing that is messing things up ... need to figure out casing issue here

Reply:
adressed

Reply:
the file should be renamed DrivetrainConfig.java.